core/modules/system/system.install

```diff
@@ -850,6 +850,9 @@ function system_install() {
   if (valid_email_address($site_mail)) {
     config_set('system.core', 'site_mail', $site_mail);
   }
+
+  // Set the install version to the current version of Backdrop.
+  config_set('system.core', 'install_version', BACKDROP_VERSION);
```

Comment:
@herbdool Yes, I can see where this would make sense. It would mean that no site would ever be listed as intalled as a dev version. The only downside I can see in this regard is possibly telemetry. But, only if we think that is useful data to have. 

I will happily make the change if specifically asked to. At this point it still an "opinion." ;-)

Reply:
It turns out `version_compare()` can handle suffixes like `dev`, `rc, `beta`, etc: https://www.php.net/version_compare

So I don't think we need to strip it off at all, we can just leave it in place and let PHP use it in the comparison.